python/mlc_chat/cli/compile.py

```diff
@@ -44,7 +44,7 @@ def _check_system_lib_prefix(prefix: str) -> str:
             "numbers (0-9), alphabets (A-Z, a-z) and underscore (_)."
         )
 
-    parser = ArgumentParser("MLC LLM Compiler")
+    parser = ArgumentParser("mlc_chat compile")
```

Comment:
This line fix the error header of help message:

```
usage: mlc_chat compile [-h] [--quantization {q0f16,q0f32,q3f16_1,q4f16_1,q4f32_1,q4f16_autoawq}] [--model-type {auto,llama,mistral,gpt2,gpt_neox,gpt_bigcode}] [--device DEVICE] [--host HOST] [--opt OPT]
                        [--system-lib-prefix SYSTEM_LIB_PREFIX] --output OUTPUT [--overrides OVERRIDES] [--debug-dump-path DEBUG_DUMP_PATH]
                        model
```

it would be the command (`mlc_chat compile`) instead of `MLC LLM Compiler`

Reply:
I see. I wasn't aware there's consistency convention here, and thanks for fixing this! Would you also like to fix other files under the `cli/` directory? Thanks!